.pre-commit-config.yaml

```diff
@@ -9,7 +9,7 @@ repos:
         args: [ --fix ]
       - id: ruff-format
   - repo: https://github.com/astral-sh/uv-pre-commit
-    rev: 0.5.18
+    rev: 0.5.20
```

Comment:
`pre-commit.ci` should run based on the hook repository version number in the PR branch, so I believe it can handle this within the PR, rather than waiting for the `push` event to occur. (This is just my assumption, as there is neither documentation nor experimentation to confirm it.)